src/js/components/stock-movement-wizard/request/AddItemsPage.jsx

```diff
@@ -370,7 +370,7 @@ class AddItemsPage extends Component {
       ) {
         lineItemsToBeUpdated.push(item);
       } else if (newQty !== oldQty || !item.quantityRequested ||
-        oldItem.comments !== item.comments) {
+        (oldItem.comments !== item.comments && _.isNil(item.comment))) {
```

Comment:
Instead of `_.isNil(item.comment)` I think it should be `_.isNil(item.comments)`. Are you sure about the `&&` here?

Reply:
@awalkowiak Yes, thanks. It should !_.isNil(item.comments), I've changed it.